karabo_data/geometry2.py

```diff
@@ -772,7 +778,11 @@ def plot_data(self, modules_data, axis_units='px', frontview=True):
 ; See: http://www.desy.de/~twhite/crystfel/manual-crystfel_geometry.html
 
 dim0 = %
-res = 5000 ; 200 um pixels
+res = {pix_size} ;
+photon_energy = 10235 ;
+clen = 0.119  ; Camera length, aka detector distance
+adu_per_eV = 0.0075  ; no idea
```

Comment:
I don't particularly like the idea of hardcoding these parameters into the geometry file.

I've been meaning for a while to make them optional arguments for `write_crystfel_geom()` so you can supply them when writing the file. If you want to tackle that as part of this PR, that would be fine. Otherwise, feel free to leave them out of this PR and we can get to it later.

Reply:
Agreed! I added some keywords to the methods. That are passed into the header!